src/python/pants/option/global_options.py

```diff
@@ -502,6 +502,12 @@ def register_options(cls, register):
     register('--lock', advanced=True, type=bool, default=True,
              help='Use a global lock to exclude other versions of pants from running during '
                   'critical operations.')
+    register('--streaming-workunits-handlers', type=list, member_type=str, default=[],
+        advanced=True,
+        help="Use this option to name Subsystems which will receive streaming workunit events. "
+        """For instance, `--streaming-workunits-handlers="['pants.reporting.workunit.Workunits']"` will """
+        """register a Subsystem called Workunits defined in the module "pants.reporting.workunit"."""
```

Comment:
Nit: don't use `"""` unless you are doing a multiline string like this:

```python
"""Line 1
Line 2
Line 3"""
```

What you are instead using is implicit string concatenation. I think you used `"""` because you need to use `"` in the string itself. Instead, what you can use is `\"`.